### PR TITLE
Fixed a few bugs

### PR DESCRIPTION
**Pull request by jm6087**
The end date was no longer populating.  Waze appeared to change the format from yyyy-mm-dd to mm/dd/yyyy
Also changed the regex to account for slashes.

### CODE REVIEW

File: WME Closure Helper Beta.user.js [hunk 1]
```diff
@@ -1122,27 +1136,34 @@ var G_AMOUNTOFPRESETS = 100;
             '<span id="wmech_lEB2h" class="wmech_closureButton wmech_lengthExtenderButton" style="background-color: #c9ffba;">+2h</span>',
             '<span id="wmech_lEB1d" class="wmech_closureButton wmech_lengthExtenderButton" style="background-color: #bafff7;">+1d</span>',
             '<span id="wmech_lEB1w" class="wmech_closureButton wmech_lengthExtenderButton" style="background-color: #bdbaff;">+1w</span>',
-            '<span id="wmech_lEB1o" class="wmech_closureButton wmech_lengthExtenderButton" style="background-color: #ffbaf9;">+1o</span>',
+            '<span id="wmech_lEB." class="wmech_closureButton wmech_lengthExtenderButton" style="background-color: #ffbaf9;">+1mo</span>',
```
Comment (SkulllzzCode) [inline WME Closure Helper Beta.user.js]:
```suggestion
            '<span id="wmech_lEB1mo" class="wmech_closureButton wmech_lengthExtenderButton" style="background-color: #ffbaf9;">+1mo</span>',
```

File: WME Closure Helper Beta.user.js [hunk 2]
```diff
@@ -1122,27 +1136,34 @@ var G_AMOUNTOFPRESETS = 100;
...
+        } else {
+        $("#wmech_lEBcustomMin").text(customCSmin + "m");
+        }
         $("#wmech_lEB1m").click(function() { addToEndStartDate(0, 0, 1); });
         $("#wmech_lEB15m").click(function() { addToEndStartDate(0, 0, 15); });
         $("#wmech_lEB1h").click(function() { addToEndStartDate(0, 0, 60); });
         $("#wmech_lEB2h").click(function() { addToEndStartDate(0, 0, 120); });
         $("#wmech_lEB1d").click(function() { addToEndStartDate(0, 1, 0); });
         $("#wmech_lEB1w").click(function() { addToEndStartDate(0, 7, 0); });
         $("#wmech_lEB1o").click(function() { addToEndStartDate(1, 0, 0); });
```
Comment (SkulllzzCode) [inline WME Closure Helper Beta.user.js]:
```suggestion
        $("#wmech_lEB1mo").click(function() { addToEndStartDate(1, 0, 0); });
```

**Review by SkulllzzCode — APPROVED** [review]
Been using past 13 days without issue.

**Review by cbrxyz — APPROVED** [review]
Thanks!

**Pull request merged**